src/components/calcite-tile-select/calcite-tile-select.scss

```diff
@@ -9,8 +9,6 @@ $spacing: $baseline/2;
   box-sizing: border-box;
   cursor: pointer;
   display: inline-block;
-  margin-bottom: 1px;
-  margin-right: -3px;
```

Comment:
We should guide users to wrap `tile-select`s inside a `tile-select-group` element, which negates the need for these hacky styles that were an attempt to get all neighboring `tile-select` elements to look right when they weren't inside a `tile-select-group` element.